app.cfg.example

```diff
@@ -134,3 +134,24 @@ poll_interval = 60
 
 # full path to the command for manipulating existing jobs
 scontrol_command = /usr/bin/scontrol
+
+# variable 'comment' under 'submitted_job_comments' should not be changed as there are regular expression patterns matching it
+[submitted_job_comments]
+description = New job on instance `{app_name}` for architecture `{arch_name}` in job dir `{symlink}` 
+comment = job id `{job_id}` awaits release by job manager 
+
+[new_job_comments]
+comment = job awaits launch by Slurm scheduler 
+
+[running_job_comments]
+comment = job `{job_id}` is running 
```

Comment:
The setting `comment` is used multiple times. It might be a bit confusing. Could we come up with unique names?